sdks/go/pkg/beam/runners/direct/direct.go

```diff
@@ -36,6 +36,12 @@ func init() {
 
 // Execute runs the pipeline in-process.
 func Execute(ctx context.Context, p *beam.Pipeline) error {
+    log.Info(ctx, "Executing pipeline with the direct runner.")
```

Comment:
I think it's a tabs vs spaces thing. Go files use Tabs for indentation, and users can configure how tabs appear spaced.
https://golang.org/doc/effective_go.html#formatting

Running gofmt handles this.

Reply:
Done, applied gofmt and squashed the fixup commit in.